metadata.json

```diff
@@ -107,6 +107,10 @@
           "id": "community",
           "name": "2.21.2 (Community)"
```

Comment:
shouldn't we add a/update to 2.23.x community version to match the 7.8 productized version?

Reply:
Well this is fuse 7.8 release, we can surely open another PR to bump community version as well...